### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Mahbuba-Nasibova**

### CODE REVIEW

**Comment by Kallen04** [comment]
Let's not merge into master yet. Keep merging into develop for now. Better if we discuss this as a team.

**Comment by Mahbuba-Nasibova** [comment]
I am trying to get rid of this empty branch...Dont know how

**Comment by Kallen04** [comment]
Which empty branch? I don't think I see it.

**Comment by Mahbuba-Nasibova** [comment]
I will send you screenshot on discord, if it is ok)

**Comment by Kallen04** [comment]
yes, send it in private message on Discord  please

**Review by inciigin — APPROVED** [review]
LGTM

File: src/utils/CalculateCost.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+     // summery cost
+    public static double sumCost(double cost1, double cost2){
+
+        return cost1 + cost2;
+
+    }
+
+    // average cost
+    public static double AvgCost(double cost1, double cost2){
```
Comment (inciigin) [inline src/utils/CalculateCost.java:13]: method name should be camel, like **avgCost**

**Review by inciigin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**